tool/debian/singa-cpp_1.0.0_amd64/DEBIAN/control

```diff
@@ -0,0 +1,12 @@
+Package: Singa
+Version: 1.0.0
+Architecture: amd64
+Maintainer: Moaz Reyad <moazreyad@gmail.com>
+Depends: libgoogle-glog-dev, libprotobuf-dev, libstdc++6 , libc6
```

Comment:
Yes. apt-get should install the needed libraries automatically. This is tested as in [SINGA-274](https://issues.apache.org/jira/browse/SINGA-274). 

p.s. the Debian files assume the build will run on amd64. If the host system used for building the Debian packages is something else, then the Architecture should be changed.